build/make_default_helpers.js

```diff
@@ -267,18 +267,24 @@ module.exports = function(docMap, config, getCurrent, Handlebars){
 			var replacer = function (match, content) {
 				var parts = content.match(linkRegExp),
 					name,
+					hashParts,
 					description,
 					linkText,
 					docObject;
 
 				name = parts ? parts[1].replace('::', '.prototype.') : content;
+				hashParts = name.split("#");
+
+				if(hashParts.length > 1){
+					name = hashParts[0];
```

Comment:
It’d be nice if there were comments so people reading this code could more easily understand what’s going on here.